src/shared/views/BuildingFootprint/FootprintDisplay.js

```diff
@@ -1,20 +1,37 @@
 // @flow
 import * as React from 'react';
-import { Image, StyleSheet, View } from 'react-native';
+import { Animated, Image, PanResponder, StyleSheet, View } from 'react-native';
+import {
+    type LayoutEvent,
+    type PressEvent,
+} from 'react-native/Libraries/Types/CoreEventTypes';
+import type {
+    GestureState,
+    PanResponderInstance,
+} from 'react-native/Libraries/Interaction/PanResponder';
 import { Path, Shape, Surface } from '@react-native-community/art';
 import tilebelt from '@mapbox/tilebelt';
+import ScaleBar from '../../common/ScaleBar';
 import type {
     BBOX,
+    ImageCoordsPoint,
+    LonLatPoint,
+    LonLatPolygon,
+    PixelCoordsPoint,
+    PixelCoordsX,
+    PixelCoordsY,
     Point,
     Polygon,
-    ProjectType,
+    SingleImageryProjectType,
     BuildingFootprintTaskType,
     Tile,
+    ZoomLevel,
 } from '../../flow-types';
 
 const GLOBAL = require('../../Globals');
 
-const tileSize = GLOBAL.SCREEN_WIDTH;
+// tileSize is only used for tile based imagery (ie: everything but google)
+const tileSize = GLOBAL.SCREEN_WIDTH / 1;
```

Comment:
why do you divide by 1?

Reply:
oops, probably some leftovers from when I was trying to adjust sizes and stuff. I guess there was a 2 there at some point.